operator/v1/types_ingress.go

```diff
@@ -282,6 +289,12 @@ var (
 	DNSReadyIngressConditionType = "DNSReady"
 )
 
+// SecuritySpec defines the schema for securing an IngressController.
+type SecuritySpec struct {
```

Comment:
Since we're in a shared operator group, are we being greedy stealing "SecuritySpec"? 😁 Would "IngressSecuritySpec" be more polite?